README.md

```diff
@@ -38,6 +38,7 @@ The NAB scores are normalized such that the maximum possible is 100.0 (i.e. the
 | [Numenta HTM](https://github.com/numenta/nupic)   | current* | 65.3             | 58.6          | 69.4          |
 | [Twitter ADVec](https://github.com/twitter/AnomalyDetection) | v1.0.0    | 47.1             | 33.6          | 53.5          |
 | [Etsy Skyline](https://github.com/etsy/skyline)  | ???     | 35.7             | 27.1          | 44.5          |
+| Bayes Changepoint          | N/A     | 17.7              | 4.2           | 33.1           |
```

Comment:
Put a footnote stating details of implementation are in nab/detectors/bayes_changept/bayes_changept_detector.py